### PR TITLE
Update webpack and most webpack deps; split bundles into 5mb each

### PR DESCRIPTION
**Pull request by christopherthielen**
screen shot 2019-02-06 at 3 34 37 pm

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
Looks like someone couldn't figure out 4MB bundles

File: package.json [hunk 1]
```diff
@@ -161,42 +161,42 @@
...
     "ts-node": "^7.0.1",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.14.0",
     "tslint-eslint-rules": "^5.3.1",
     "tslint-loader": "^3.6.0",
     "tslint-react": "^3.6.0",
     "typescript": "^3.2.4",
-    "uglifyjs-webpack-plugin": "^1.2.4",
-    "url-loader": "^1.0.1",
+    "uglifyjs-webpack-plugin": "2.1.1",
```
Comment (anotherchrisberry) [inline package.json]: Do we still need this if we are using terser?

**Comment by christopherthielen** [comment]
UglifyJS is no longer maintained.  Terser works on ES6 and is the default in webpack 5. 

Splitting to 5mb chunks seems like an OK idea, but will cause more initial fetches on app start.  Thoughts?

File: yarn.lock [hunk 2]
```diff
@@ -739,6 +765,159 @@
...
+    "@webassemblyjs/ast" "1.7.11"
+    "@webassemblyjs/helper-wasm-bytecode" "1.7.11"
+    "@webassemblyjs/ieee754" "1.7.11"
+    "@webassemblyjs/leb128" "1.7.11"
+    "@webassemblyjs/utf8" "1.7.11"
+
+"@webassemblyjs/wasm-opt@1.7.11":
+  version "1.7.11"
+  resolved "https://registry.yarnpkg.com/@webassemblyjs/wasm-opt/-/wasm-opt-1.7.11.tgz#b331e8e7cef8f8e2f007d42c3a36a0580a7d6ca7"
+  integrity sha512-XynkOwQyiRidh0GLua7SkeHvAPXQV/RxsUeERILmAInZegApOUAIJfRuPYe2F7RcjOC9tW3Cb9juPvAC/sCqvg==
```
Comment (anotherchrisberry) [inline yarn.lock]: nice

File: package.json (hunk 1 above)
Comment (christopherthielen) [reply package.json]: nope, good catch

File: yarn.lock [hunk 3]
```diff
@@ -3182,6 +3131,15 @@ chalk@^2.4.1:
     escape-string-regexp "^1.0.5"
     supports-color "^5.3.0"
 
+chalk@^2.4.2:
+  version "2.4.2"
+  resolved "https://registry.yarnpkg.com/chalk/-/chalk-2.4.2.tgz#cd42541677a54333cf541a49108c1432b44c9424"
+  integrity sha512-Mti+f9lpJNcwF4tWV8/OrTTtF1gZi+f8FqlyAdouralcFWFQWF2+NgCHShjkCb+IFBLq9buZwE1xckQU4peSuQ==
+  dependencies:
+    ansi-styles "^3.2.1"
+    escape-string-regexp "^1.0.5"
```
Comment (anotherchrisberry) [inline yarn.lock]: yes

File: yarn.lock (hunk 2 above)
Comment (christopherthielen) [reply yarn.lock]: thx

**Comment by sbwsg** [comment]
lgtm!

File: yarn.lock (hunk 3 above)
Comment (christopherthielen) [reply yarn.lock]: indeed

**Pull request merged**